charts/prometheus-elasticsearch-exporter/ci/extra-manifests.yaml

```
@@ -0,0 +1,12 @@
+---
+# Create extra manifests
+
+extraManifests:
+  - |
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      labels:
+        name: prometheus-elasticsearch-exporter-extra
```

Apologies, it is really too late. Name is missing, e.g.
```suggestion
    metadata:
      name: prometheus-elasticsearch-exporter-extra       
```